### PR TITLE
pull_request#1_that_contains_the_3_features_of_sprint2

### PR DESCRIPTION
**Pull request by G-jonathan**
Merge request which contains the features provided in sprint 2:
--> creation of a new "develop" branch
--> add the packages corresponding at the MVC architecture(controller,repository,model,service)
--> configuration of the “application.properties" file mainly for log levels and the activation of the H2 database console
--> write the h2 database creation script in resources/data.sql
--> Modeling tables as a java entity: model/MedicalRecords.java, model/Firestations.java, model/Persons.java

### CODE REVIEW

File: src/main/resources/data.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS persons;
+DROP TABLE IF EXISTS firestations;
+DROP TABLE IF EXISTS medicalrecords;
+
+CREATE TABLE persons (
```
Comment (Asavarkhul) [inline src/main/resources/data.sql:5]: Hmm I think that a table name should be singular -> `PERSON`

File: src/main/java/com/openClassroomsProject/SafetyNetAlerts/model/Persons.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.openClassroomsProject.SafetyNetAlerts.model;
+
+import lombok.Data;
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name="persons")
+public class Persons {
```
Comment (Asavarkhul) [inline src/main/java/com/openClassroomsProject/SafetyNetAlerts/model/Persons.java:9]: This is a class -> Singular name, you are not returning a collection inside this class/

`Person`

**Review by Asavarkhul — APPROVED** [review]

File: src/main/java/com/openClassroomsProject/SafetyNetAlerts/model/Persons.java (hunk 2 above)
Comment (G-jonathan) [reply src/main/java/com/openClassroomsProject/SafetyNetAlerts/model/Persons.java:9]: yes you're right

File: src/main/resources/data.sql (hunk 1 above)
Comment (G-jonathan) [reply src/main/resources/data.sql:5]: yes you're right

**Pull request merged**